pkg/api/blackduck/v1/types.go

```diff
@@ -46,6 +46,7 @@ type BlackduckView struct {
 	ContainerTags     []string          `json:"containerTags"`
 	Version           string            `json:"version"`
 	SupportedVersions []string          `json:"supportedVersions"`
+	NodeAffinities    string            `json:nodeAffinities`
```

Comment:
pls replace `json:nodeAffinities` to `json:"nodeAffinities"`

Reply:
done